tests/test_formatter.py

```diff
@@ -38,12 +38,18 @@
 except LookupError:
     testDataDirectory = None
 
+try:
+    hitsDirectory = lsst.utils.getPackageDir("ap_verify_ci_hits2015")
```

Comment:
This package is not in the table file so won't be set up. I assume that's deliberate since it's likely to be huge? If we are testing a formatter can we instead copy the relevant file(s) into testdata_decam?

Reply:
Yeah, this is something I've been struggling with. The problem is that the calibrations I'm using to test from `ap_verify_ci_hits2015` are 1.3GB, so we can't just copy them into testdata_decam. I could try the trick I used for testdata_jointcal (zeroing the data and compressing), but that makes it harder to test against the expected gen2 values.

A worse problem (which is not immediately apparent here) is in the use of `ci_hits2015` for the convert test: it needs an existing gen2 repo, but `ci_hits2015` doesn't come with a gen2 repo, so the path I have there doesn't exist in `ci_hits2015`. I'm very open to ideas on what to do here: I needed a way to run DECam convert tests and this gets me that locally, but it doesn't really help with CI overall.

Reply:
Note: I've filed DM-23828 to deal with the question of where to put the gen2 repo we need for proper testing. We'll live with this in its current state, but make it a priority to fix that so that we are running these tests as part of CI.